secrets/external.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 SAP SE
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package secrets
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func CheckPWCMD(pwCmd string) (string, error) {
```

Comment:
```suggestion
func CheckPwCmdEnv() {
```
we should hardcode the env into this function otherwise we might need to update each implementation when something about that env changes which we are trying to avoid here.

Also I suggested a different name for the function to better reflect what it does but if you have an even better idea, please raise it.

I've changed the return type because we probably want to use `os.SetEnv` to continue using gopercloud/utils/clientconfig like before and don't require any code changes.
Also with the change off using logg.fatal in the end, we wouldn't return any error.